src/components/MaterialUI/Inputs/Radio.js

```diff
@@ -50,9 +50,9 @@ const extractAndValidateProps = radioProps => {
 	const defaultVal = radioProps.get(RadioProps.propNames.defaultVal);
 	const value = radioProps.get(RadioProps.propNames.value) ?? defaultVal;
 	const update = radioProps.get(RadioProps.propNames.update);
-	const row = radioProps.get(RadioProps.propNames.row) || true;
+	const row = radioProps.get(RadioProps.propNames.row) ?? true;
```

Comment:
The || could be used, as any false-like value will then be set to true.

Reply:
@TONYFECTEAU row property is **boolean**, if true render group as row, if false render verticaly. 
When you assign row=false - you will have anyway true.
So correct is to use ?? operator, so if it is undefined or null, then it is true.